docs/builtins.rst

```diff
@@ -373,3 +373,13 @@ Filter accepts several arguments:
 * indentation character: character to be used for indentation. Default is a space.
 * indent first line: whether first line of output should be indented or not. Default is ``false``.
 
+``filter``
+~~~~~~~~~
+
+Applies filter by its name, provided as an argument.
```

Comment:
Applies the filter with the name (provided as argument) to the current expression.